data/campaigns/Eastern_Invasion/units/Human_Terraent_Paladin.cfg

```diff
@@ -38,12 +61,6 @@
             halo=units/terraent/terraent-halo[1~12].png:150
         [/frame]
     [/standing_anim]
-    [movement_anim]
-        [frame]
-            image="units/terraent/terraent[1~4,2,5].png:[200,300*3,200*2]"
-            halo=units/terraent/terraent-halo[1~12].png:150
-        [/frame]
-    [/movement_anim]
```

Comment:
Why is the movement_anim removed?

Reply:
Original has 2 movement_anims - this one (which seems to be just a clone of the standing_anim), and another that's just the baseframe.  I don't think it matters much which one is used, but it seems wrong to have two.